packages/cli/src/modules/data-table/data-store-column.repository.ts

```diff
@@ -40,7 +41,11 @@ export class DataStoreColumnRepository extends Repository<DataTableColumn> {
 			});
 
 			if (existingColumnMatch) {
-				throw new DataStoreColumnNameConflictError(schema.name, dataTableId);
+				const dataTable = await em.findOneBy(DataTable, { id: dataTableId });
+				if (!dataTable) {
+					throw new UnexpectedError('Data store not found');
```

Comment:
this shouldn't be needed - the data.store.service.ts addColumn call to this method has

		`await this.validateDataStoreExists(dataStoreId, projectId);`

check on it - was there some case where you ran into this?